uwblueprint/community-fridge-kw
# fix: csv export changes

## Brief description. What is this change? 

### [CSV Export Changes](https://www.notion.so/CSV-Export-Changes-b97cfa2c8fb54e9799fe56ed7de31e03)


## Implementation description. How did you make this change?
* Modified the csv to only show true values for booleans and if null or false, then it should be blank in the csv
* Move isPickup and pickupLocation after volunteerNeeded for the columns in export food rescues


## Steps to test
1. Download food rescues
2. Check that only true values are shown
3. Check that the ordering of the columns is correct

## Checklist
